### PR TITLE
Add VS Code integration

### PR DESCRIPTION
**Pull request by flying-sheep**
This makes it easier to teach this as everything is already set up when people use VS Code. Maybe the variable should be a multiple choice for “none”, “vs code”, “all”, with the ability to extend support for PyCharm and so on?

### CODE REVIEW

File: {{cookiecutter.project_name}}/.gitignore [hunk 1]
```diff
@@ -3,6 +3,13 @@
 *~
 buck-out/
 
+{% if not cookiecutter.ide_integration -%}
```
Comment (Zethson) [inline {{cookiecutter.project_name}}/.gitignore:6]: Out of convenience, I would just always add these to the `.gitignore`. Even if the person setting up the project might not be using them, other contributors will.

File: {{cookiecutter.project_name}}/.gitignore [hunk 2]
```diff
@@ -3,6 +3,13 @@
...
+# IDEs
+/.idea/
```
Comment (Zethson) [inline {{cookiecutter.project_name}}/.gitignore:8]: This is a very technical detail that is not important but there are a few files in the `.idea` folder that Jetbrains thinks are fine to commit. But let's ignore this.

File: {{cookiecutter.project_name}}/.gitignore (hunk 1 above)
Comment (Zethson) [reply {{cookiecutter.project_name}}/.gitignore:6]: Oh wait I think I got the logic the wrong way around. Now we're keeping them, right?

**Review by Zethson — APPROVED** [review]

File: {{cookiecutter.project_name}}/.gitignore (hunk 1 above)
Comment (flying-sheep) [reply {{cookiecutter.project_name}}/.gitignore:6]: I always commit them actually.

File: {{cookiecutter.project_name}}/.gitignore (hunk 2 above)
Comment (flying-sheep) [reply {{cookiecutter.project_name}}/.gitignore:8]: yeah, that’s what I mean with

> Maybe the variable should be a multiple choice for “none”, “vs code”, “all”, with the ability to extend support for PyCharm and so on?

**Pull request merged**